### PR TITLE
migrate to RN 0.61.*

### LINKED ISSUE #9470: Ignore Dark Mode setting on iOS13
# Problem

At present, whenever dark mode is enabled on iOS, it only introduces a bunch of bugs without any advantages. 

- Status bars are presented white making them invisible against our white backgrounds, 

- photo picker dialogues are presented with incorrect styling all over, making them difficult to use
- dark keyboards are an unwelcome sight without a Dark Mode of our own.

## Implementation

To quote the maestro @flexsurfer 

> it seems like simple one to ignore dark mode
add this to your Info.plist file:
    <key>UIUserInterfaceStyle</key>
    <string>Light</string>

## Acceptance Criteria

There are no visible changes to our UI when dark mode is enabled.

thank you kindly! 🙏

### PR DESCRIPTION
**Pull request by rasom**
This change allows to fix status bar on iOS in dark mode #9470

status: ready
fix #9470

### CODE REVIEW

File: figwheel-bridge.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ var config = {
 var React = require('react');
 var createReactClass = require('create-react-class');
 var ReactNative = require('react-native');
-var WebSocket = require('WebSocket');
+//var WebSocket = WebSocket; //require('WebSocket');
```
Comment (flexsurfer) [inline figwheel-bridge.js]: could you elaborate please?  why do we comment this and how will it work

**Review by flexsurfer — APPROVED** [review]

**Review by errorists — APPROVED** [review]
judging by the amount of Pedro comments it's the good stuff. tested dark mode, it's fully ignored now as it should! thanks @rasom 🙏

File: figwheel-bridge.js (hunk 1 above)
Comment (rasom) [reply figwheel-bridge.js]: It is a global var from now on, I will just remove this line

File: ios/Podfile [hunk 2]
```diff
@@ -9,26 +9,36 @@ target 'StatusIm' do
...
   pod 'React-cxxreact', :path => '../node_modules/react-native/ReactCommon/cxxreact'
   pod 'React-jsi', :path => '../node_modules/react-native/ReactCommon/jsi'
   pod 'React-jsiexecutor', :path => '../node_modules/react-native/ReactCommon/jsiexecutor'
   pod 'React-jsinspector', :path => '../node_modules/react-native/ReactCommon/jsinspector'
-  pod 'yoga', :path => '../node_modules/react-native/ReactCommon/yoga'
+  pod 'ReactCommon/jscallinvoker', :path => "../node_modules/react-native/ReactCommon"
+  pod 'ReactCommon/turbomodule/core', :path => "../node_modules/react-native/ReactCommon"
+  pod 'Yoga', :path => '../node_modules/react-native/ReactCommon/yoga'
+
+  pod 'DoubleConversion', :podspec => '../node_modules/react-native/third-party-podspecs/DoubleConversion.podspec'
```
Comment (pedropombeiro) [inline ios/Podfile]: @rasom This pod is already listed below, no need to add it again.

Comment (rasom) [reply ios/Podfile]: sure

**Comment by rasom** [comment]
@churik can you please check if auto test failures are related to this PR

File: ios/Podfile (hunk 2 above)
Comment (rasom) [reply ios/Podfile]: fixed

File: figwheel-bridge.js (hunk 1 above)
Comment (rasom) [reply figwheel-bridge.js]: fixed

**Comment by churik** [comment]
@rasom, no e2e failures are not related to this PR for sure

**Review by jakubgs — APPROVED** [review]

**Review by yenda — APPROVED** [review]

**Pull request merged**